### PR TITLE
Fix #220 ignore virtualbox docker volume

### LINKED ISSUE #220: Make docker volume group optional in dev
Needed for ubuntu

### PR DESCRIPTION
**Pull request by robnagler**
Just use the main disk for docker.

### CODE REVIEW

File: rsconf/package_data/base_os/main.sh.jinja [hunk 1]
```diff
@@ -57,28 +57,6 @@ base_os_logical_volume() {
...
-                install_err "$bdev contains mounted partitions; destroy with VirtualBox"
-            fi
-            if pvck "$bdev"; then
-                install_err "physical volume $bdev already initialized; destroy with VirtualBox"
-            fi
-            pvcreate /dev/sdb
-            vgcreate docker /dev/sdb
-        fi
-    fi
     {{ base_os.logical_volume_cmds }}
```
Comment (e-carlin) [inline rsconf/package_data/base_os/main.sh.jinja:60]: This results in an empty bash func if there are no cmds.

**Review by e-carlin — CHANGES_REQUESTED** [review]

File: rsconf/package_data/base_os/main.sh.jinja (hunk 1 above)
Comment (robnagler) [reply rsconf/package_data/base_os/main.sh.jinja:61]: Good catch 852639e

**Comment by e-carlin** [comment]
Blocked from running due to https://github.com/radiasoft/download/issues/313

**Comment by e-carlin** [comment]
Worked around the other issue.

**Comment by e-carlin** [comment]
lgtm. I'll let you merge since I'm not sure about all the changes to the tests and whether that was intended for this PR

**Review by e-carlin — APPROVED** [review]

**Pull request merged**